po/de.po

```diff
@@ -5733,7 +5728,7 @@ msgstr "Architektur festlegen"
 
 #: /tmp/fish.i8YroE/implicit/share/completions/apt-file.fish:13
 msgid "Set sources.list file"
-msgstr "sources.list-Datei angeben"
+msgstr "Sources.list-Datei festlegen"
```

Comment:
I think this actually has to be named `sources.list`, maybe put it in quotes?

Reply:
That was me following the linter to blindly :D